week2/prep-exercises/1-blog-API/server.js

```diff
@@ -1,10 +1,57 @@
-const express = require('express')
-const app = express();
- 
+const express = require('express');
+const fs = require('fs');
 
-// YOUR CODE GOES IN HERE
+const app = express();
+app.use(express.json());
 app.get('/', function (req, res) {
-  res.send('Hello World')
-})
- 
-app.listen(3000)
\ No newline at end of file
+  res.send('Hello World');
+});
+
+function camelize(str) {
+  return str
+    .replace(/(?:^\w|[A-Z]|\b\w)/g, function (word, index) {
+      return index === 0 ? word.toLowerCase() : word.toUpperCase();
+    })
+    .replace(/\s+/g, '');
+}
+
+app.get('/posts/', (req, res) => {
```

Comment:
```suggestion
app.get('/posts/:title', (req, res) => {
```

Normally got `GET` requests for a single resource we specify the unique identifier in the path to fetch it. There isn't a body provided for these requests.